.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: go
 
 go:
-  - 1.11.x
-  - 1.12.x
   - 1.13.x
+  - 1.14.x
```

Comment:
The testing library we use dropped support for older ones. https://github.com/stretchr/testify#supported-go-versions

Side Note - I don't see it in our README, but i think we should explicitly mention the supported versions as well.

Reply:
We really should do the side note on all of our SDKs

Reply:
one less to do : https://rubygems.org/gems/smartcar B-)